packages/azure-themes/src/azure/styles/TeachingBubble.styles.ts

```diff
@@ -0,0 +1,56 @@
+import { ITeachingBubbleStyleProps, ITeachingBubbleStyles } from 'office-ui-fabric-react/lib/TeachingBubble';
+import { BaseColors } from '../AzureColors';
+import { Depths } from '../AzureDepths';
+import * as StyleConstants from '../Constants';
+
+export const TeachingBubbleStyles = (props: ITeachingBubbleStyleProps): Partial<ITeachingBubbleStyles> => {
+  const { theme } = props;
+  const { palette, semanticColors } = theme;
+
+  return {
+    bodyContent: {
+      color: semanticColors.bodyText,
+      border: `1px solid ${palette.themePrimary}`
+    },
+    footer: {
+      color: semanticColors.bodyText
+    },
+    header: {
+      color: semanticColors.bodyText
+    },
+    headline: {
+      color: semanticColors.bodyText
+    },
+    content: {
+      backgroundColor: semanticColors.bodyBackground,
+      color: semanticColors.bodyText
+    },
+    subText: {
+      color: semanticColors.bodyText
+    },
+    subComponentStyles: {
+      callout: {
+        root: {
+          borderColor: semanticColors.inputBorder,
+          borderStyle: StyleConstants.borderSolid,
+          borderWidth: StyleConstants.borderWidth,
+          boxShadow: Depths.depth8
+        }
+      }
+    },
+    closeButton: {
+      color: semanticColors.bodyText,
+      margin: '1px',
```

Comment:
```suggestion
      margin: 1,
```

Reply:
We too are concerned about selector specificity.  We've had to get rather 'artful' in some case.  But there may be better ways to do the same thing.  Do you have a couple of standouts you encountered.  Anything dealing with buttons generally caught our attention.